workflow/rocoto/gefs_tasks.py

```diff
@@ -219,11 +219,6 @@ def _atmosoceaniceprod(self, component: str):
             dep_dict = {'type': 'metatask', 'name': 'fcst_mem#member#'}
             deps.append(rocoto.add_dependency(dep_dict))
             dependencies = rocoto.create_dependency(dep=deps, dep_condition='or')
-        elif component in ['ice']:
-            command = f"{self.HOMEgfs}/ush/check_ice_netcdf.sh @Y @m @d @H #fhr# &ROTDIR; #member# {fhout_ice_gfs}"
-            dep_dict = {'type': 'sh', 'command': command}
-            deps.append(rocoto.add_dependency(dep_dict))
-            dependencies = rocoto.create_dependency(dep=deps)
```

Comment:
Should ice be added to the `if` rather than the `else`so it doesn't begin pre-maturely?

Reply:
You are right. After looking at this further, it looks like there is no need for an if-statement, since the atmos_prod, ocean_prod and ice_prod dependencies are structured the same way (all checking for the existence of a history file or the completion of a fcst metatask). I set this up similar to how the ocn/ice/atmos prod dependencies are set up in `workflow/rocoto/gfs_tasks.py`. These changes have been pushed.